config/deploy/stage.rb

```diff
@@ -1,5 +1,6 @@
-server 'pres-robots1-stage.stanford.edu', user: 'pres', roles: %w[web app db]
+server 'preservation-robots1-stage.stanford.edu', user: 'pres', roles: %w[web app db]
 
 Capistrano::OneTimeKey.generate_one_time_key!
 
 set :deploy_environment, 'test'
+set :default_env, robot_environment: fetch(:deploy_environment)
```

Comment:
:default_env was set in deploy.rb;  if it didn't work there, it should be removed.

Reply:
good catch